src/components/Post/index.js

```diff
@@ -43,6 +81,24 @@ function Post({
 				postImage.current.style.maxHeight = "350px";
 			}
 		};
+    // Getting Post's like data and updating like state
+    db.collection("posts")
+    .doc(postId)
+    .collection("likes")
+    .onSnapshot((snap) => {
+      let documents = [];
+      snap.forEach((doc) => {
+        documents.push({ userLiked: doc.id });
```

Comment:
Since you are storing users who have liked a doc, you can also display them, by clicking the `no. of likes` button. I would suggest using a _modal_ to display them. Further, instead of `doc.id` store `doc.displayName`.